test/functional/api/api-3rd-party-add.bats

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bats
+
+# Author: Castulo Martinez
+# Email: castulo.martinez@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	# Skip this test for local development because we write the certificate on /
+	# This is necessary because the default certificate location is hardcoded on build time and we
+	# need to run swupd without -C parameter to test this feature.
+	if [ -z "${RUNNING_IN_CI}" ]; then
+		skip "Skipping test because it will make changes to your system"
+	fi
+
+	create_test_environment "$TEST_NAME"
+
+	if [ ! -f /usr/share/clear/update-ca/Swupd_Root.pem ]; then
```

Comment:
You can't guarantee that a test like this will run fine when running in parallel with other tests. Move that to test/functional/only_in_ci_system/